JSDemos/Demos/TreeList/BatchEditing/description.md

```diff
@@ -1 +1,10 @@
-Batch editing is&nbsp;very similar to&nbsp;cell editing in&nbsp;that a&nbsp;user edits data cell by&nbsp;cell. The only difference is&nbsp;that with batch editing a&nbsp;user needs to&nbsp;click the _&laquo;Save changes&raquo;_ button to&nbsp;actually save changes. Only then the TreeList will collect all changes in&nbsp;a&nbsp;batch and send them to&nbsp;the data source.
\ No newline at end of file
+You can use the batch edit mode to defer saving multiple changes. In this mode, users edit data one cell at a time. The changes are stored in a buffer and can be discarded before the **Save** button is clicked.
+
+To enable the batch edit mode, configure the following properties:
+
+* Set **editing**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) to *"batch"*.
+* Assign **true**  to the [editing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowDeleting) properties.
+
+This demo uses the [onInitNewRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onInitNewRow) handler to populate cells of a new row with default values. This function sets *"Not Started"* as the initial task status and specifies the current date as the initial start and due dates.
+
+If you store data on the server and have a custom backend logic to apply data changes, implement the [onSaving](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onSaving) client-side function to save all changes with a single request as done in the DataGrid's [Batch Update Request](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/BatchUpdateRequest/) demo.
```

Comment:
We already have a perfectly fine paragraph in the Batch Update Request description:

```suggestion
If data is stored on a server, our TreeList sends multiple requests to save edited objects - one request per object (this is because most servers only process one edit operation at a time). If your server supports batch update, you can configure the TreeList to save all changes with a single request. Refer to the following demo for more information: [DataGrid: Batch Update Request]((https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/BatchUpdateRequest/).
```